xModuleAnalyzer-NET.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# For use with linux actions in https://github.com/whitesource-ft/ws-examples/tree/main/Prioritize/DotNet/Multi-Module
+
+for csproject in  $(find ./src -type f \( -wholename "*.csproj" ! -wholename "*build*" ! -wholename "*test*" ! -wholename "*host*" ! -wholename "*migration*" \))
+
+do
+echo "Found" $csproject 
+CSPROJ=$(basename $csproject .csproj)
+
+find ./ -type f \( -wholename "*/bin/Release/netcore*/$CSPROJ.dll" ! -wholename "*build*" ! -wholename "*test*" ! -wholename "*host*" ! -wholename "*migration*" \) -print >> multi-module.txt
```

Comment:
dll is not the only possible extension for the appPath, it could also be exe.
Plus, you should probably remove the /netcore* part from the path pattern, because not all projects are necessarily built by .NET Core

Reply:
Are .exe files created from Linux builds?  I wouldn't assume this would happen but would love to see an example to help improve the script.  Agreed on the netcore part.  Will remove it as I see some netstandard instances happening, as well.